### ISSUE TITLE
The sequence of Keypoints

### ISSUE THREAD

**Issue by ShiningSord** [shubham-goel/4D-Humans#45]
Hi, 

Thanks for your powerful work and released code. It is so cool!

When I tried to use this code, I found that the HMR keypoint detector in your demo outputs 44 keypoints for each person.

However, 44 KP is not a regularly used setting in 2D pose estimation. 

Can you tell me the meaning of each keypoint?

Thanks for your attention.

**Comment by geopavlakos** [comment]
We mostly adopt the order from SPIN - we use the 25 keypoints listed [here](https://github.com/nkolot/SPIN/blob/1e7d8ff4cb0705cf04211760c875098da553c1f8/constants.py#L14), and the first 19 from the 24 keypoints listed [here](https://github.com/nkolot/SPIN/blob/1e7d8ff4cb0705cf04211760c875098da553c1f8/constants.py#L41), so we end up with 44 keypoints.

**Comment by ShiningSord** [comment]
Thanks for your kind reply! It solve my problem perfectly.
I close this issue now.

**Comment by Justinitaa** [comment]
@geopavlakos Hi, I have a doubt, according to you, there should be duplicates of 44 joints in here, for example, 'OP RShoulder': 17 and 'OP RShoulder': 17 in the figure below, are they counted as the same joint repeated twice? If they are counted as the same joint, then there should be duplicates of the 44 joint coordinates in the 3d_joints output in the .pkl file, but I checked that there are no duplicates in there. How do I get a one-to-one correspondence between the 3d_joints information and the joints in the .pkl file output in the figure below?
![屏幕截图 2024-11-06 171659](https://github.com/user-attachments/assets/4b5b588f-c380-4700-8f8c-1cbcaf8f8d8d)
For example, here is the 3d_joints part of the information I extracted inside the .pkl file
![Uploading 屏幕截图 2024-11-06 172925.png…]()
